src/routes.rs

```diff
@@ -66,3 +71,16 @@ async fn health(data: web::Data<Mutex<AppData>>) -> impl Responder {
         temp: temperature,
     })
 }
+
+#[get("/slots")]
+async fn get_slots(data: web::Data<Mutex<AppData>>) -> impl Responder {
+    let slots = machine::get_slots(data.lock().unwrap().config.clone());
+    let temp = machine::get_temperature(data.lock().unwrap().config.clone());
+
+    let temp = temp * (9.0/5.0) + 32.0;
```

Comment:
If we're redesigning, can we just report celsius?

Reply:
Sure